docker/Dockerfile

```diff
@@ -10,15 +10,15 @@ COPY requirements.txt /requirements.txt
 RUN YARL_NO_EXTENSIONS=1 MULTIDICT_NO_EXTENSIONS=1 pip install --no-cache-dir --prefix=/install -r /requirements.txt
 
 # Load and verify Cosign
-FROM debian:buster-slim as cosign_loader
+FROM debian:bullseye-slim as cosign_loader
```

Comment:
this is only the "builder" image for Cosign. Did you intend to modify that?
While there is very low risk in using a somewhat vulnerable build image, I'd presume the fix is aimed at the actual connaisseur image below. Is that correct?
In fact, Connaisseur uses alpine (`python:3-alpine` see line 1) as a base as shown in line 28 below.

Reply:
ah yes... just tried a new build. Thanks! I'll close.

Reply:
I actually think updating the build image is still valuable. Will reopen and adjust quickly :slightly_smiling_face: